erroranalysis/erroranalysis/_internal/surrogate_error_tree.py

```diff
@@ -49,8 +49,31 @@ def compute_json_error_tree(analyzer,
                             max_depth=DEFAULT_MAX_DEPTH,
                             num_leaves=DEFAULT_NUM_LEAVES,
                             min_child_samples=DEFAULT_MIN_CHILD_SAMPLES):
-    # Note: this is for backcompat for older versions
-    # of raiwidgets pypi package
+    """Computes the error tree for the given dataset.
+
+    Note: this is for backcompat for older versions
+    of raiwidgets pypi package
+
+    :param analyzer: The error analyzer containing the categorical
+        features and categories for the full dataset.
+    :type analyzer: BaseAnalyzer
+    :param features: The features to train the surrogate model on.
+    :type features: numpy.array or pandas.DataFrame
```

Comment:
Is this numpy.ndarray or numpy.array?

import numpy as np
np.ndarray
<class 'numpy.ndarray'>
np.array

Seems like numpy.array is built-in function array.

Reply:
done, replaced numpy.array with numpy.ndarray everywhere